src/utils/generateRandomColor.ts

```diff
@@ -0,0 +1,11 @@
+import randomSeed from 'random-seed';
+
+export function generateRandomColor(seed: string): string {
```

Comment:
When you have a single function in a module named after that function, I think it makes more sense to export it as the `default` export from the module.